### ISSUE TITLE
Nvidia Login Fails since the latest update

### ISSUE THREAD

**Issue by pabloptechedge** [hmlendea/gfn-electron#125]
### Describe the bug

The login completes in the popup window. but it never seems to go through to the main window.

I assume this was because of some change on nvidia's end

### To Reproduce

Log out out of Nvidia and log back in again

### Operatingsystem

Ubuntu 20.04 LTS

### Desktop Environment

Gnome

### Installation method

snap

### Version

1.5.1

**Comment by massung** [comment]
Same issue for me (on Mint 20.04 LTS).

**Comment by theRoboxx** [comment]
Linux Mint 21.3 (Ubuntu 20.04 LTS): same issue with npm and flatpak.
Version 1.5.1
Terminal output for `npm start`:

```
> geforcenow-electron@1.5.1 start
> electron --enable-accelerated-mjpeg-decode --enable-accelerated-video --ignore-gpu-blacklist --enable-native-gpu-memory-buffers --enable-gpu-rasterization .

Using user agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.84 Safari/537.36
(electron) Deprecation Warning: Disabling nativeWindowOpen is deprecated. The nativeWindowOpen option will be removed in Electron 18.
[13385:0505/213638.098062:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
(node:13356) UnhandledPromiseRejectionWarning: Error: Could not connect
    at Socket.onerror (/home/Stewart/Desktop/gfn-electron/node_modules/discord-rpc/src/transports/ipc.js:32:16)
    at Object.onceWrapper (node:events:510:26)
    at Socket.emit (node:events:390:28)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
(Use `electron --trace-warnings ...` to show where the warning was created)
(node:13356) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
```

Terminal output for `flatpak run io.github.hmlendea.geforcenow-electron`:
```
Using user agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.84 Safari/537.36
Gtk-Message: 21:07:00.840: Failed to load module "xapp-gtk3-module"
[2:0505/210700.987175:ERROR:bus.cc(397)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0505/210700.987247:ERROR:bus.cc(397)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
(electron) Deprecation Warning: Disabling nativeWindowOpen is deprecated. The nativeWindowOpen option will be removed in Electron 18.
(node:2) UnhandledPromiseRejectionWarning: Error: Could not connect
    at Socket.onerror (/app/geforcenow-electron/resources/app.asar/node_modules/discord-rpc/src/transports/ipc.js:32:16)
    at Object.onceWrapper (node:events:510:26)
    at Socket.emit (node:events:390:28)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
(Use `geforcenow-electron --trace-warnings ...` to show where the warning was created)
(node:2) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
[39:0505/210701.340868:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
```

**Comment by PC-Heini** [comment]
+1

**Comment by hmlendea** [comment]
Related to #126  

Should be fixed in [v1.5.2](https://github.com/hmlendea/gfn-electron/releases/tag/v1.5.2) (by #127)

**Comment by jjardon** [comment]
Confirm is fixed in v1.5.2
